### DISCUSSION TITLE
Γραφικά

### DISCUSSION THREAD

**Discussion by inf2021090** [ContattoContare/site#26] [Ideas]
@inf2021229 έχουμε παρόμοιες θεματικές μπορούμε να τα συνδυάσουμε σε ενότητες: χρονολόγιο-γραφικά υπολογιστών και διαφάνειες-προσομοιώσεις.

**Comment by NikosTrypakis** [comment]
> @inf2021229 έχουμε παρόμοιες θεματικές μπορούμε να τα συνδυάσουμε σε ενότητες: χρονολόγιο-γραφικά υπολογιστών και διαφάνειες-προσομοιώσεις.

Έχεις δίκιο και πιστεύω πως είναι καλή ιδέα αφού είναι αρκετά κοντά οι επιλογές μας.
Μπορούμε να κάνουμε ακόμα ένα slide για τον σχεδιασμό γραφικών κάτι το οποίο νομίζω βολεύει και τους δύο μας.
